src/main_chat/response_continuation.py

```diff
@@ -0,0 +1,41 @@
+import configuration
+import src.common_utils.language_utils.statement_utils as statement
+from src.common_utils.database.database_service import DatabaseProxy
+from src.common_utils.language_utils.sentence_filter_utils import SentenceFilter
+
+
+def initialize_db_with_continue_statements():
+    # fixme: to be removed after adding to database initialization file
+
+    db = DatabaseProxy('mongodb://localhost:27017/', 'PepperChatDB')
```

Comment:
please pass db as class constructor parameter as it is done in many chatbot components which use database